include/deal.II/dofs/dof_handler.h

```diff
@@ -25,6 +25,7 @@
 #include <deal.II/base/iterator_range.h>
 #include <deal.II/base/std_cxx11/shared_ptr.h>
 #include <deal.II/dofs/block_info.h>
+#include <deal.II/dofs/dof_accessor.h>
```

Comment:
Do you really need this in the header file? Do we have inline functions where we actually dereference iterators in the header file?